src/avio.jl

```diff
@@ -631,8 +631,11 @@ if have_avdevice()
         camera_devices = String[]
 
         read_vid_devs = false
-        out, err = readall_stdout_stderr(`$ffmpeg -list_devices true -f $idev -i $idev_name`)
-        buf = length(out) > 0 ? out : err
+        #out, err = readall_stdout_stderr(`$ffmpeg -list_devices true -f $idev -i $idev_name`)
+        #buf = length(out) > 0 ? out : err
+        
+        out = open(`$ffmpeg -list_devices true -f $idev -i $idev_name`, other=devnull)
```

Comment:
`other` is a positional argument, not a keyword argument.

Reply:
My mistake, now corrected

Reply:
After seeing the latest error, I remembered that `avio.jl` redefines `open` without importing `Base.open` (on purpose).  You'll need to call this with `Base.open()`